### ISSUE TITLE
switch.js: switch page add vlan failed if 2 or more switch exist

### ISSUE THREAD

**Issue by ptpt52** [openwrt/luci#5714]
/etc/board.json
```
...
        "switch": {
                "switch0": {
                        "enable": true,
                        "reset": true,
                        "ports": [
                                {
                                        "num": 0,
                                        "role": "lan"
                                },
                                {
                                        "num": 1,
                                        "role": "lan"
                                },
                                {
                                        "num": 2,
                                        "role": "lan"
                                },
                                {
                                        "num": 3,
                                        "role": "lan"
                                },
                                {
                                        "num": 4,
                                        "role": "wan"
                                },
                                {
                                        "num": 6,
                                        "device": "eth0",
                                        "need_tag": false,
                                        "want_untag": false
                                }
                        ],
                        "roles": [
                                {
                                        "role": "lan",
                                        "ports": "0 1 2 3 6t",
                                        "device": "eth0.1"
                                },
                                {
                                        "role": "wan",
                                        "ports": "4 6t",
                                        "device": "eth0.2"
                                }
                        ]
                },
                "switch1": {
                        "enable": true,
                        "reset": true,
                        "ports": [
                                {
                                        "num": 0,
                                        "role": "lan"
                                },
                                {
                                        "num": 1,
                                        "role": "lan"
                                },
                                {
                                        "num": 2,
                                        "role": "lan"
                                },
                                {
                                        "num": 3,
                                        "role": "lan"
                                },
                                {
                                        "num": 5,
                                        "device": "eth1",
                                        "need_tag": false,
                                        "want_untag": false
                                }
                        ],
                        "roles": [
                                {
                                        "role": "lan",
                                        "ports": "0 1 2 3 5t",
                                        "device": "eth1.1"
                                }
                        ]
                }
        },

...
```

switch.js not work property if 2 or more switch dev exist

**Comment by ptpt52** [comment]
<img width="1088" alt="xx" src="https://user-images.githubusercontent.com/5331426/157489813-d9117487-46df-41c8-8f51-e53710fd2623.png">

**Comment by ptpt52** [comment]
@jow-  hi
could you help fix this issue?

**Comment by ptpt52** [comment]
switch display wrong:
config/network:
```
config switch
        option name 'switch0'
        option reset '1'
        option enable_vlan '1'

config switch_vlan
        option device 'switch0'
        option vlan '1'
        option ports '0 1 2 3 6t'

config switch_vlan
        option device 'switch0'
        option vlan '2'
        option ports '4 6t'

config switch
        option name 'switch1'
        option reset '1'
        option enable_vlan '1'

config switch_vlan
        option device 'switch1'
        option vlan '1'
        option ports '0 1 2 3 5t'

```

vlan 2 missing on switch0 on the page.

**Comment by jow-** [comment]
Fixed by #5718
